src/Uno.Extensions.Maui.UI/Extensibility/Impl/MauiEmbeddingExtension.cs

```diff
@@ -120,13 +129,12 @@ public void InitializeMauiEmbeddingApp(MauiApp mauiApp, Application app)
 		var scope = mauiApp.Services.CreateScope();
 		var rootContext = new MauiContext(scope.ServiceProvider, androidApp);
 		InitializeScopedServices(rootContext);
-
 		var iApp = mauiApp.Services.GetRequiredService<IApplication>();
 		_ = new MauiEmbedding.EmbeddedApplication(mauiApp.Services, iApp);
 
 		// Initializing with the Activity to set the current activity.
 		// The Bundle is not actually used by Maui
-		Microsoft.Maui.ApplicationModel.Platform.Init(activity, null);
+		//Microsoft.Maui.ApplicationModel.Platform.Init(activity, null);
```

Comment:
Any reason to keep this comment around, as opposed to just removing it plus the above comment block?